scripts/storybook/runner.js

```diff
@@ -0,0 +1,83 @@
+// @ts-check
+
+/**
+ * This is a start-storybook wrapper that automatically pre-builds dependencies that are needed to run storybook,
+ * then invokes `start-storybook` so CLI experience is the same as running raw `start-storybook` binary
+ *
+ *
+ * Note:
+ * - This is used only for vNext packages
+ * - We cannot leverage build-less experience unfortunately at the moment. To have it we will need a bigger overhaul of "how we build" things.
+ *
+ * @example
+ *
+ * ```sh
+ * node ./scripts/storybook/runner
+ * ```
+ */
+
+const path = require('path');
+const fs = require('fs');
+const { execSync } = require('child_process');
+const chalk = require('chalk');
+
+main();
+
+function main() {
+  const args = process.argv.slice(2);
+  const COMMAND_PREFIX = `${chalk.cyan('>')} ${chalk.inverse(chalk.bold(chalk.cyan(' STORYBOOK RUNNER ')))}`;
+  const implicitDependencies = [
+    {
+      name: '@fluentui/react-storybook-addon',
+      description: 'fluentui storybook addon that adds functionality to storybook',
+    },
+  ];
+  const dependencies = [
+    { name: '@fluentui/babel-make-styles', description: 'custom babel plugin that compiles make-styles definitions' },
+  ];
+
+  /**
+   * @type {{name:string;devDependencies: Record<string,string>,[key:string]:unknown}}
```

Comment:
> reading the JSON is a bit simpler/easier 

I try to avoid any unnecessary abstraction/tiny packages for simple things that are build within the platform